### PR TITLE
[wheel] provide a valid default commit

### PR DESCRIPTION
**Pull request by aslonnie**
and only build one wheel, aligning with CI's behavior.

### CODE REVIEW

**Review by can-anyscale — APPROVED** [review]
should we add tests for this workflow if we still need to maintain this; this basically can break any time ;( (e.g. ugprade manylinux)

**Comment by aslonnie** [comment]
> should we add tests for this workflow if we still need to maintain this; this basically can break any time ;( (e.g. ugprade manylinux)

added `BUILD_ONE_PYTHON_ONLY` too, so that it is closer to what runs in CI.

**Comment by aslonnie** [comment]
> should we add tests for this workflow if we still need to maintain this; this basically can break any time ;( (e.g. ugprade manylinux)

and we can change this to a locally runnable wrapper script or something.. maybe in a follow up.

**Comment by aslonnie** [comment]
cc @brucez-anyscale

File: python/README-building-wheels.md [hunk 1]
```diff
@@ -9,10 +9,15 @@ produce .whl files owned by root.
 Inside the root directory (i.e., one level above this python directory), run
 
 ```
-docker run -e TRAVIS_COMMIT=<commit_number_to_use> --rm -w /ray -v `pwd`:/ray -ti quay.io/pypa/manylinux2014_x86_64  /ray/python/build-wheel-manylinux2014.sh
+docker run -ti --rm \
+    -e BUILDKITE_COMMIT="$(git rev-parse HEAD)" \
+    -e BUILD_ONE_PYTHON_ONLY=py39 \
+    -w /ray -v `pwd`:/ray \
+    quay.io/pypa/manylinux2014_x86_64 \
+    /ray/python/build-wheel-manylinux2014.sh
```
[the remark this answers is not in the document]
Comment (aslonnie) [reply python/README-building-wheels.md:17]: done.

**Review by brucez-anyscale — APPROVED** [review]

**Pull request merged**